package.json

```diff
@@ -80,7 +80,7 @@
     "ng-packagr": "^3.0.1",
     "prismjs": "^1.12.2",
     "protractor": "~5.3.1",
-    "rxjs": "6.0.0",
+    "rxjs": "6.2.2",
```

Comment:
Note that updating the version of rxjs to 6.2.2 is needed for the fake async test with observables to pass.
The following commit in rxjs fixes the problem:
https://github.com/ReactiveX/rxjs/commit/01eefff2132f29dfe51476782c3e7f2db48f88f2

Reply:
I guess the ideal solution is for the fix to be released in `rxjs 6.0.x`... I see three options in order of preference:
- use a workaround if any
- `src/pacakge.json` → `rxjs: ^6.0.0`, `package.json` → `rxjs: 6.2.2`. And avoid using features from `6.1` and `6.2` →  `endWith` operator and `isObservable` methods https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#620-2018-05-22
- `src/pacakge.json` → `rxjs: ^6.2.2`, `package.json` → `rxjs: 6.2.2`